Revit_Core_Adapter/Revit_Core_Adapter.csproj

```diff
@@ -12,6 +12,7 @@
     <AssemblyName>Revit_Core_Adapter_2018</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <TargetFrameworkProfile />
```

Comment:
Please revert the changes in this file, it is irrelevant to this PR (you can e.g. switch back to `master` for a second, copy the file somewhere, then switch back to this branch and paste the file).